### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by QuillzLouis** [etiennepinchon/hologram#11]
Linus support would be really nice
(If anyone has alternatives for a while. pls tell me)

**Comment by karanganesan** [comment]
@QuillzLouis I think @etiennepinchon can make a Linux Build too it's easy only it's a Electron based app ! 😊

**Comment by QuillzLouis** [comment]
@karanganesan Yeah being the dummo I am knowing it's electron I tried to compile it spoiler doesn't work lol.

**Comment by QuillzLouis** [comment]
Still no progress?

**Comment by QuillzLouis** [comment]
Would love to hear an update from @etiennepinchon

**Comment by etiennepinchon** [comment]
Hey dude 👋 
I have no plan for Linux just yet but who knows.. 
I let you know if anything changes ;)

**Comment by QuillzLouis** [comment]
@etiennepinchon Not even windows support, not all of us can afford macs :/

**Comment by petrk94** [comment]
I agree @QuillzLouis 
Mac is nice, but expensive, even in Virtual Box the image is not running smoothly, so at least Linux were nice. Windows is not a major thing, but Linux should be possible, please :)

**Comment by petrk94** [comment]
@QuillzLouis @etiennepinchon any update on this nearly one year old issue?

**Comment by shantee** [comment]
Still no news how to make a build for linux .. that's such a pity  it looks really great !

**Comment by petrk94** [comment]
@shantee 
In theory it should work 
I checked the build and just need npm as its running on JS, so its just a lack of the developer in packaging in other formats.
But I dont have tested it, if it work.
